src/components/Dialogs/GeneralDialog/GeneralDialog.style.ts

```diff
@@ -4,16 +4,18 @@ import { Button } from '@/shared/components'
 import { colors, sizes, breakpoints } from '@/shared/theme'
 
 export const StyledContainer = styled.div`
+  --dialog-padding-small: ${sizes(4)};
+  --dialog-padding-large: ${sizes(6)};
```

Comment:
We can actually simplify this! This lets us share the values easily but one problem still persists - whatever consumes this data still has to know what breakpoint to use. So there's still some unnecessary coupling between the components - there's some knowledge that you need to have.

One of the awesome things CSS vars allow is actually this: instead of the styles being responsive and changing which value they use, the value itself can be responsive:

```suggestion
  --dialog-padding: ${sizes(4)};
  @media screen and (min-width: ${breakpoints.small}) {
    --dialog-padding: ${sizes(6)};
  }
```

Then in any consumer we don't have to include the media query but just consume the value which will change automatically